platform/darwin/src/MGLVectorSource.h

```diff
@@ -42,10 +43,41 @@ NS_ASSUME_NONNULL_BEGIN
 MGL_EXPORT
 @interface MGLVectorSource : MGLTileSource
 
+#pragma mark Initializing a Source
+
 - (instancetype)initWithIdentifier:(NSString *)identifier configurationURL:(NSURL *)configurationURL NS_DESIGNATED_INITIALIZER;
 
 - (instancetype)initWithIdentifier:(NSString *)identifier tileURLTemplates:(NS_ARRAY_OF(NSString *) *)tileURLTemplates options:(nullable NS_DICTIONARY_OF(MGLTileSourceOption, id) *)options NS_DESIGNATED_INITIALIZER;
 
+#pragma mark Querying the source's features
```

Comment:
Since jazzy uses `mark`s to generate section headings, it’s important to keep them consistent (title case, describes a task, uses the most commonplace terms possible). How about “Accessing a Source’s Content”, which is the same `mark` we use in MGLShapeSource?